### PR TITLE
Gremlin Dotnet Windows Build fixes

### PR DESCRIPTION
**Pull request by L0Lmaker**
*Issue #, if available:*
https://bitquill.atlassian.net/jira/software/c/projects/AN/boards/36?selectedIssue=AN-1147

*Description of changes:*
1
```
[ERROR] Failed to execute goal org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:execute (update-version) on project gremlin-dotnet-source: Error occurred while calling a method on a Groovy class from classpath.: InvocationTargetException: startup failed:
[ERROR] Script1.groovy: 2: unexpected char: '\' @ line 2, column 24.
[ERROR]    def file = new File("C:\Users\rithink\Repos\L0Lmaker\tinkerpop-copy-001\gremlin-dotnet\src/Gremlin.Net/Gremlin.Net.csproj")
[ERROR]                           ^
[ERROR]
[ERROR] 1 error
[ERROR] -> [Help 1]
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs [hunk 1]
```diff
@@ -119,7 +122,11 @@ public async Task ShouldUseSpecifiedEvaluationTimeout()
                 evaluationStopWatch.Stop();
                 Assert.Contains("ServerTimeout", thrownException.Message);
                 Assert.Contains(timeOutInMs.ToString(), thrownException.Message);
-                Assert.True(evaluationStopWatch.ElapsedMilliseconds < scriptSleepTimeInMs);
+                Console.WriteLine(evaluationStopWatch.ElapsedMilliseconds);
```
Comment (vkagamlyk) [inline gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs]: remove debug code

Comment (lyndonbauto) [reply gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs]: But what is someone wanted to know if 'im here!!!'? ;)

Comment (L0Lmaker) [reply gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs]: oops will do

Comment (L0Lmaker) [reply gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs]: removed in [9f5ec9a](https://github.com/lyndonbauto/tinkerpop/pull/113/commits/9f5ec9a20564c7be5422f55c6c5b269281d9e754)

**Review by vkagamlyk — APPROVED** [review]

**Review by lyndonbauto — APPROVED** [review]

**Review by vkagamlyk — APPROVED** [review]

**Review by vkagamlyk — COMMENTED** [review]
LGTM
Please squash before open PR to apache/

**Pull request merged**